.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -23,7 +23,14 @@ body:
         * Set defaults on (or omit) any variables. The person reproducing it should not need to invent variable settings
         * If multiple steps are required, such as running tofu twice, consider scripting it in a simple shell script. Providing a script can be easier than explaining what changes to make to the config between runs.
         * Omit any unneeded complexity: remove variables, conditional statements, functions, modules, providers, and resources that are not needed to trigger the bug
-
+  - type: textarea
+    id: community-note
+    attributes:
+      label: Community note
+      description: Please leave this note unchanged.
+      value: |
+        > [!TIP]
+        > 👋 Hi there, OpenTofu community! The OpenTofu team prioritizes issues based on upvotes. Please make sure to upvote this issue and describe how it affects you in detail in the comments if you would like to see it implemented.
```

Just some suggestion on the wording.
```suggestion
        > 👋 Hi there, OpenTofu community! The OpenTofu team prioritizes issues based on upvotes. Please make sure to upvote this issue and describe how it affects you in detail in the comments if you want to have this pushed higher on the priority list of the OpenTofu team.
```

---

Same for the other file.

---

Agreed with @yottta, this rewording would make it clearer on how we prioritize.